src/backend/routers/public.ts

```diff
@@ -0,0 +1,7 @@
+import { publicProcedure, router } from "../trpc";
+
+export const publicRouter = router({
+  healthCheck: publicProcedure.query(() => {
+    return "Ok";
+  }),
```

Comment:
To clarify: This is a placeholder for a health check, right? It doesn't actually check anything. I'm able to hit it with `http://localhost:3000/api/trpc/public.healthCheck` on my local, and it returns an successful response even if I've stopped the database and minio container.

It might be worth adding a comment to note that this is just a placeholder. Or maybe it's not needed as it's obvious to everyone who's not new to trpc like me. 😄

Reply:
@mrabbitt yes, this is just to verify the api is up and running to it at least checks that we are receiving and responding to api calls so I don't consider it just a placeholder. great callout to have it do more, though. I thought about a quick db call, but hadn't considered minio ... although we don't want it to take too long as this would be an endpoint we would configure with hosting providers that might alert us when the site is down, hence the need for it to be a public route. but most of all I wanted to introduce a public api route so that the excludeList in the test would be put to use and could be verified for correctness. thanks!